# insomnia
Codes for integrative genomics analysis of insomnia-associated genes

# Tittle
Integrative genomics analysis identifies 5 promising genes implicated in insomnia risk based on multiple omics datasets

# Abstract 

  In recent decades, many genome-wide association studies on insomnia have reported numerous genes harboring multiple risk variants. Nevertheless, the molecular functions of these risk variants conveying risk to insomnia are still ill-studied. In this study, we integrated GWAS summary statistics (N = 386,533) with two independent brain expression quantitative trait loci (eQTL) datasets (N = 329) to determine whether expression-associated SNPs convey risk to insomnia. Furthermore, we applied numerous bioinformatics analyses to highlight promising genes associated with insomnia risk. By using Sherlock integrative analysis, we detected 449 significant insomnia-associated genes in the discovery stage. These identified genes were significantly overrepresented in 6 biological pathways including Huntington’s disease (P = 5.58×10-5), Alzheimer’s disease (P = 5.58×10-5), Parkinson’s disease (P = 6.34×10-5), spliceosome (P = 1.17×10-4), oxidative phosphorylation (P = 1.09×10-4), and wnt signaling pathways (P = 2.07×10-4). Further, 5 of these identified genes were replicated in an independent brain eQTL dataset. Through a PPI network analysis, we found that there existed highly functional interactions among these 5 identified genes. 3 genes of LDHA (P = 0.044), DALRD3 (P = 5.0×10-5) and HEBP2 (P = 0.032) showed significantly lower expression level in brain tissues of insomnic patients than that in controls. In addition, the expression levels of these 5 genes showed prominently dynamic changes across different time points between behavioral states of sleep and sleep deprivation in mice brain cortex. Together, the evidence of this study strongly suggested that these 5 identified genes may represent candidate genes and contributed risk to the etiology of insomnia.


